### PR TITLE
Update pending subscription notice [MAILPOET-5631]

### PR DESCRIPTION
**Pull request by johnolek**
## Description

This PR updates the pending approval notices throughout the plugin with new copy and to all be consistent.

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[MAILPOET-5631](https://mailpoet.atlassian.net/browse/MAILPOET-5631)

## After-merge notes

_N/A_

## Tasks

- [x] I followed [best practices](https://codex.wordpress.org/I18n_for_WordPress_Developers) for translations
- [x] I added sufficient test coverage
- [x] I embraced TypeScript by either creating new files in TypeScript or converting existing JavaScript files when making changes

[MAILPOET-5631]: https://mailpoet.atlassian.net/browse/MAILPOET-5631?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by johnolek** [comment]
@NeosinneR Feel free to take a look at this one yourself if you're so inclined.

File: mailpoet/lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php [hunk 1]
```diff
@@ -220,27 +226,13 @@ private function getEmailVolumeLimitReachedMessage(): string {
...
-  }
-
   /**
    * Returns error $message and $operation for API::RESPONSE_CODE_CAN_NOT_SEND
    */
   private function getCanNotSendError(array $result, $sender): array {
     if ($result['error'] === API::ERROR_MESSAGE_PENDING_APPROVAL) {
       $operation = MailerError::OPERATION_PENDING_APPROVAL;
-      $message = $this->getPendingApprovalMessage();
+      $message = $this->pendingApprovalNotice->getPendingApprovalMessage();
```
Comment (brezocordero) [inline mailpoet/lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php]: Nitpick: Not sure if this makes any difference but the original message from `getPendingApprovalMessage()` had a `<br/>` appended at the end

**Review by brezocordero — DISMISSED** [review]
@johnolek all good, nice refactoring!
Just a small nitpick, feel free to ignore and pass to QA.

File: mailpoet/lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php (hunk 1 above)
Comment (johnolek) [reply mailpoet/lib/Mailer/Methods/ErrorMappers/MailPoetMapper.php]: Good catch! I'm not sure myself how important it is, but we may as well keep it consistent. I added it back in 1c8cc23bd795a23cda272011b19fda574e32c64d.

**Review by brezocordero — DISMISSED** [review]
LGTM!
Thanks for making the change.

**Comment by Aschepikov** [comment]
Hi @johnolek there are some conflicts can you resolve them pls

**Review by Aschepikov — APPROVED** [review]
GJ!

**Pull request merged**